common/OpTestUtil.py

```diff
@@ -1082,6 +1082,23 @@ def ping_fail_check(self, i_ip):
             return False
         return True
 
+    def check_pci_type(self, pci_device, cv_HOST):
+        print("SACHIN_DEBUG4 : from utils", pci_device)
+        self.bus_id = pci_device[0:7]
+        search_string = "OF_TYPE=network"
```

Comment:
search_string is never used .. do you need it

Reply:
Done